packages/commonwealth/client/scripts/views/modals/NewCommunityAdminModal/NewCommunityAdminModal.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+
+import { formatAddressShort } from 'helpers';
+import { uniqBy } from 'lodash';
+import app from 'state';
+import { CWText } from 'views/components/component_kit/cw_text';
+import { SelectedCommunity } from 'views/components/component_kit/new_designs/CWCommunitySelector';
+import {
+  CWModalBody,
+  CWModalFooter,
+  CWModalHeader,
+} from 'views/components/component_kit/new_designs/CWModal';
+import { CWSelectList } from 'views/components/component_kit/new_designs/CWSelectList';
+import { CWButton } from 'views/components/component_kit/new_designs/cw_button';
+import './NewCommunityAdminModal.scss';
+
+interface NewCommunityAdminModalProps {
+  onModalClose: () => void;
+  handleClickConnectNewWallet: () => void;
+  handleClickContinue: (selectedAddress: string) => void;
+  selectedCommunity: SelectedCommunity;
+}
+
+const NewCommunityAdminModal = ({
+  onModalClose,
+  handleClickConnectNewWallet,
+  handleClickContinue,
+  selectedCommunity,
+}: NewCommunityAdminModalProps) => {
+  const availableAddressesOnSelectedChain = app.user.addresses.filter(
```

Comment:
I think we should add "?." checks here

Reply:
yeah I can do this, however, I dont think it is necessary as `app.user.addresses` is empty array by default.